poetry/masonry/builders/sdist.py

```diff
@@ -168,10 +168,17 @@ def build_setup(self):  # type: () -> bytes
             before.append("extras_require = \\\n{}\n".format(pformat(extras)))
             extra.append("'extras_require': extras_require,")
 
-        entry_points = self.convert_entry_points()
+        entry_points, scripts = self.convert_entry_points()
         if entry_points:
             before.append("entry_points = \\\n{}\n".format(pformat(entry_points)))
             extra.append("'entry_points': entry_points,")
+        if scripts:
+            before.append(
+                "scripts = \\\n{}\n".format(
+                    pformat([f"bin/{name}" for name in scripts])
```

Comment:
Can you replace that f-string with old-style string formatting, so as to support older versions of Python?